alws/crud.py

```diff
@@ -250,9 +250,29 @@ async def modify_distribution(build_id: int, distribution: str, db: Session,
             await pulp_client.modify_repository(remove=value, repo_to=key)
 
 
+async def sign_start(
+            db: Session, 
+            build_task_id: int
+        ) -> models.BuildTask:
+    async with db.begin():
+        ts_expired = datetime.datetime.now() - datetime.timedelta(minutes=20)
+        query = models.BuildTask.id == build_task_id
```

Comment:
I think, that doesn\`t look like query, may be `where_condition` or something else?
Also, you can assign query in variable then use it in db.execute()
```python
query = select(...).where(...).with_for_update()
db_task = await db.execute(query)
```

Reply:
This query works. I checked it. Or I misunderstood you.